client/app/components/catalogs/_catalogs.sass

```diff
@@ -0,0 +1,2 @@
+.catalog-save-buttons
```

Comment:
rather than creating a new class, recommend using an existing one, `.pull-right` does just this thing.